app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,7 @@
 
 <nav id="header" class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
     <!-- ||| menu, displayed for mobile: -->
-  <a class="navbar-brand ml-lg-5" id="brand" href="<%= root_path %>">Public Lab</a>
+  <a class="navbar-brand ml-lg-5" id="brand" href="<%= root_path %>">Home</a>
```

Comment:
Hey @okonek , thanks so much for opening this...I am sorry for wrong context the issue was changed and description was not changed...I have updated the issue. We would like to leave the text as is for now but add a tooltip that says home...something like..
`  <a class="navbar-brand ml-lg-5" id="brand" href="<%= root_path %> data-toggle= "tooltip" title="Home">Public Lab</a> `.
Thanks